met-web/src/components/engagement/listing/AdvancedSearch/SearchComponent.tsx

```diff
@@ -115,7 +115,7 @@ const AdvancedSearch: React.FC<filterParams> = ({ setFilterParams }) => {
         >
             <Grid item xs={12} lg={2}>
                 <FormControl component="fieldset">
-                    <MetLabel>Status</MetLabel>
+                    <BodyText bold>Status</BodyText>
```

Comment:
Are we replacing all instances of MetLabel with BodyText in the future? Is it a cosmetic change?